json/jira-index-pattern.json

```diff
@@ -5,7 +5,7 @@
-                "release_date": "2019-03-06T10:46:46.695412",
+                "release_date": "2019-03-20T14:54:28.379195",
```

Comment:
As nothing changed but the export date, could you please remove it from the commit?